cli/js/web/console.ts

```diff
@@ -96,19 +99,146 @@ function createIterableString<T>(
   }
   ctx.add(value);
 
-  const entries: string[] = [];
+  let entries: string[] = [];
   // In cases e.g. Uint8Array.prototype
   try {
     for (const el of value) {
       entries.push(config.entryHandler(el, ctx, level + 1, maxLevel));
     }
   } catch (e) {}
   ctx.delete(value);
+  if(entries.length > MAX_ARRAY_LENGTH){
+    const nmore = entries.length - MAX_ARRAY_LENGTH;
+    entries = entries.slice(0, MAX_ARRAY_LENGTH);
+    entries.push(`... ${nmore} more items`);
+  }
   const iPrefix = `${config.displayName ? config.displayName + " " : ""}`;
-  const iContent = entries.length === 0 ? "" : ` ${entries.join(", ")} `;
+  let iContent;
+  if(config.group){
+    iContent = groupEntries(entries, level, value);
+    const indetation = "  ".repeat(level + 1);
+    if(iContent.length > 1){
+      iContent = `\n${indetation}${iContent.join(`,\n${indetation}`)}\n${"  ".repeat(level)}`;
+    }
+  }else{
+    iContent = entries.length === 0 ? "" : ` ${entries.join(", ")} `;
+    if(iContent.length > LINE_BREAKING_LENGTH){
+      iContent = `\n${" ".repeat(level + 1)}${entries.join(`,\n${" ".repeat(level+1)}`)}\n`;
+    }
+  }
   return `${iPrefix}${config.delims[0]}${iContent}${config.delims[1]}`;
 }
 
+
+// Sliglhly modified node.js groupArrayElements. Remeber to include copyrights
```

Comment:
```suggestion
// Slightly modified node.js groupArrayElements. Remember to include copyrights.
```

But we should include copyright instead of just mentioning this.